Server/DiscordBot.cs

```diff
@@ -1,160 +1,386 @@
+    //        SettingsLoadHandler();
+    //        Logger.Info(
+    //            $"Discord bot logged in as {DiscordClient.CurrentUser.Username}#{DiscordClient.CurrentUser.Discriminator}");
+    //        Reconnecting = false;
+    //        string mentionPrefix = $"{DiscordClient.CurrentUser.Mention}";
+    //        DiscordClient.MessageCreated += async (_, args) => {
+    //            if (args.Author.IsCurrent) return; //dont respond to commands from ourselves (prevent "sql-injection" esq attacks)
+    //            //prevent commands via dm and non-public channels
+    //            if (CommandChannel == null) {
+    //                if (args.Channel is DiscordDmChannel)
+    //                    return; //no dm'ing the bot allowed!
+    //            }
+    //            else if (args.Channel.Id != CommandChannel.Id && (LogChannel != null && args.Channel.Id != LogChannel.Id))
+    //                return;
+    //            //run command
+    //            try {
+    //                DiscordMessage msg = args.Message;
+    //                string? resp = null;
+    //                if (string.IsNullOrEmpty(Prefix)) {
+    //                    await msg.Channel.TriggerTypingAsync();
+    //                    resp = string.Join('\n', CommandHandler.GetResult(msg.Content).ReturnStrings);
+    //                } else if (msg.Content.StartsWith(Prefix)) {
+    //                    await msg.Channel.TriggerTypingAsync();
+    //                    resp = string.Join('\n', CommandHandler.GetResult(msg.Content[Prefix.Length..]).ReturnStrings);
+    //                } else if (msg.Content.StartsWith(mentionPrefix)) {
+    //                    await msg.Channel.TriggerTypingAsync();
+    //                    resp = string.Join('\n', CommandHandler.GetResult(msg.Content[mentionPrefix.Length..].TrimStart()).ReturnStrings);
+    //                }
+    //                if (resp != null)
+    //                {
+    //                    foreach (string mesg in SplitMessage(resp))
+    //                        await msg.RespondAsync(mesg);
+    //                }
+    //            } catch (Exception e) {
+    //                Logger.Error(e);
+    //            }
+    //        };
+    //        DiscordClient.ClientErrored += (_, args) => {
+    //            Logger.Error("Discord client caught an error in handler!");
+    //            Logger.Error(args.Exception);
+    //            return Task.CompletedTask;
+    //        };
+    //        DiscordClient.SocketErrored += (_, args) => {
+    //            Logger.Error("This is probably that stupid bug again!");
+    //            Logger.Error("Discord client caught an error on socket!");
+    //            Logger.Error(args.Exception);
+    //            return Task.CompletedTask;
+    //        };
+    //    } catch (Exception e) {
+    //        Logger.Error("Exception occurred in discord runner!");
+    //        Logger.Error(e);
+    //    }
+    //}
+    #endregion
```

Comment:
Make sure to comment out the old code once you are ready to merge

Reply:
Yeah I've kept it around for reference. I'll delete it once I'm done

Reply:
*remove, my bad